client/src/components/Connections/Connections.css

```diff
@@ -0,0 +1,28 @@
+.Connections__heading {
+  font-size: 2rem;
+  font-family: var(--default-font);
+  color: var(--purple);
+}
+
+.Connections__list {
+  display: flex;
+  width: 100%;
+}
+
+.Connections__link {
+  color: var(--blue-lite);
+  font-size: 2rem;
+  font-family: var(--default-font);
+  font-weight: normal;
+}
+
+.Connections__person {
+  margin: 0 2rem 0 0;
+}
+
+.Connections--name {
+  font-size: 2rem;
+  font-family: var(--default-font);
+  text-align: center;
+}
+
```

Comment:
Make your component responsive! You can reduce the number of users that appear on a smaller screen and also wrap the users so that two appear in the column with the `extraUsers` message at the end.